src/test/java/seedu/address/testutil/TypicalTeachingAssistant.java

```diff
@@ -82,10 +83,43 @@
             .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
             .build();
 
+    //===Contact===
+    public static final Contact AVA = new ContactBuilder().withName("Ava Pauline")
```

Comment:
thank you for this

Reply:
thanks for appreciating, had to think of names starting with alphabets a, b, c, d, e, f, g, h, i that weren't already used :')